tasks/src/tasksData.ts

```diff
@@ -18,42 +18,51 @@ import textClassification from "./text-classification/data";
 import textGeneration from "./text-generation/data";
 
 
-// To make comparisons easier, task order is the same as in /lib/interfaces/Types.ts
+// To make comparisons easier, task order is the same as in PIPELINE_TAGS_DISPLAY_ORDER in /lib/interfaces/Types.ts
```

Comment:
```suggestion
// To make comparisons easier, task order is the same as in PIPELINE_DATA in /lib/interfaces/Types.ts
```
same suggestion here